From 866f0953895cc8a9f99a2a15f8fbfd655e621e0e Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Wed, 1 Sep 2021 21:35:33 +0200
Subject: [PATCH] Don't add dashes to version

---
 build/build.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/build/build.py b/build/build.py
index ae512b9..6e3eeaa 100755
--- a/build/build.py
+++ b/build/build.py
@@ -209,10 +209,10 @@ def get_bazel_path(bazel_path_flag):
 
 def check_bazel_version(bazel_path):
   try:
-    version_output = shell([bazel_path, "--version"])
+    version_output = shell([bazel_path, "version"])
   except subprocess.CalledProcessError:
     return False
-  match = re.search(r"bazel *([0-9\\.]+)", version_output)
+  match = re.search(r"Build label: *([0-9\\.]+)", version_output)
   if match is None:
     return False
   actual_ints = [int(x) for x in match.group(1).split(".")]
-- 
2.33.0

