From 866f0953895cc8a9f99a2a15f8fbfd655e621e0e Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Wed, 1 Sep 2021 21:35:33 +0200
Subject: [PATCH] Don't add dashes to version

---
--- jax-jaxlib-v0.1.74.orig/build/build.py	2021-11-16 22:17:01.000000000 -0300
+++ jax-jaxlib-v0.1.74/build/build.py	2021-12-09 14:37:18.138160554 -0300
@@ -204,10 +204,10 @@
 
 def get_bazel_version(bazel_path):
   try:
-    version_output = shell([bazel_path, "--version"])
+    version_output = shell([bazel_path, "version"])
   except subprocess.CalledProcessError:
     return None
-  match = re.search(r"bazel *([0-9\\.]+)", version_output)
+  match = re.search(r"Build label: *([0-9\\.]+)", version_output)
   if match is None:
     return None
   return tuple(int(x) for x in match.group(1).split("."))
-- 
2.33.0

