From bbdf2dde4f3a32a2265efcb9f7382dd985e7cadc Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwelk@xhochy.com>
Date: Sun, 8 Oct 2023 19:34:34 +0200
Subject: [PATCH 1/2] Allow for custom CUDA build

---
 build/build.py | 14 ++++++++++----
 1 file changed, 10 insertions(+), 4 deletions(-)

diff --git a/build/build.py b/build/build.py
index d38b911bb..932587ba8 100755
--- a/build/build.py
+++ b/build/build.py
@@ -573,6 +573,13 @@ async def main():
 
     if args.cuda_version:
       logging.debug("Hermetic CUDA version: %s", args.cuda_version)
+      # set GCC_HOST_COMPILER_PATH for toolchain with conda-forge
+      wheel_build_command_base.append(
+          f"--action_env=GCC_HOST_COMPILER_PATH=\"{os.environ['GCC']}\""
+      )
+      wheel_build_command_base.append(
+          f"--action_env=GCC_HOST_COMPILER_PREFIX=\"{os.path.dirname(os.environ['GCC'])}\""
+      )
       wheel_build_command_base.append(
           f"--repo_env=HERMETIC_CUDA_VERSION={args.cuda_version}"
       )
@@ -687,10 +694,9 @@ async def main():
 
         wheel_build_command.append(f"--jaxlib_git_hash={git_hash}")
 
-      result = await executor.run(wheel_build_command.get_command_as_string(), args.dry_run, args.detailed_timestamped_log)
-      # Exit with error if any wheel build fails.
-      if result.return_code != 0:
-        raise RuntimeError(f"Command failed with return code {result.return_code}")
+      import subprocess
+      subprocess.check_call(wheel_build_command.get_command_as_string(),
+                            shell=True)
 
   # Exit with success if all wheels in the list were built successfully.
   sys.exit(0)
