From e03061546d4b95610465b22227e36ebcdd36c728 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwe.korn@quantco.com>
Date: Thu, 18 Jul 2024 15:55:38 +0200
Subject: [PATCH] Add x64_windows as a wheel platform

---
 build/build.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build/build.py b/build/build.py
index 0c25cea..db4a2a4 100755
--- a/build/build.py
+++ b/build/build.py
@@ -575,6 +575,7 @@ def main():
       "darwin_x86_64": "x86_64",
       "ppc": "ppc64le",
       "aarch64": "aarch64",
+      "x64_windows": "x64_windows",
   }
   # TODO(phawkins): support other bazel cpu overrides.
   wheel_cpu = (wheel_cpus[args.target_cpu] if args.target_cpu is not None
-- 
2.39.3 (Apple Git-146)

