From c850b31214595ff30a3612f9f95369685f6f0bb5 Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwelk@xhochy.com>
Date: Sun, 8 Oct 2023 19:34:34 +0200
Subject: [PATCH 1/4] Allow for custom CUDA build

---
 build/build.py | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/build/build.py b/build/build.py
index 2b4f71d..653204c 100755
--- a/build/build.py
+++ b/build/build.py
@@ -701,10 +701,9 @@ async def main():
       if wheel =="jax":
         wheel_build_command.append("--//jax:build_jaxlib=false")
 
-      result = await executor.run(wheel_build_command.get_command_as_string(), args.dry_run, args.detailed_timestamped_log)
-      # Exit with error if any wheel build fails.
-      if result.return_code != 0:
-        raise RuntimeError(f"Command failed with return code {result.return_code}")
+      import subprocess
+      subprocess.check_call(wheel_build_command.get_command_as_string(),
+                            shell=True)
 
   output_path = args.output_path
   jax_bazel_dir = os.path.join("bazel-bin", "dist")
