From 95d25f8f01b3ad2a006040c8348d16684767932e Mon Sep 17 00:00:00 2001
From: "Uwe L. Korn" <uwelk@xhochy.com>
Date: Sun, 8 Oct 2023 19:34:34 +0200
Subject: [PATCH 1/2] Allow for custom CUDA build

---
 build/build.py | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/build/build.py b/build/build.py
index ccb1e63..739f881 100755
--- a/build/build.py
+++ b/build/build.py
@@ -247,6 +247,11 @@ def write_bazelrc(*, python_bin_path, remote_build,
       f.write("build --action_env TF_CUDA_PATHS=\"{tf_cuda_paths}\"\n"
               .format(tf_cuda_paths=",".join(tf_cuda_paths)))
     if cuda_version:
+      # set GCC_HOST_COMPILER_PATH for toolchain with conda-forge
+      f.write("build --action_env GCC_HOST_COMPILER_PATH=\"{gcc_host_compiler_path}\"\n"
+              .format(gcc_host_compiler_path=os.environ["GCC"]))
+      f.write("build --action_env GCC_HOST_COMPILER_PREFIX=\"{gcc_host_compiler_prefix}\"\n"
+              .format(gcc_host_compiler_prefix=os.path.dirname(os.environ["GCC"])))
       f.write("build --action_env TF_CUDA_VERSION=\"{cuda_version}\"\n"
               .format(cuda_version=cuda_version))
     if cudnn_version:
-- 
2.39.3 (Apple Git-145)

