From 295e968fca6184dd4b412bf3be3a0402ab82e8cc Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Mon, 24 Feb 2025 07:59:42 +1100
Subject: [PATCH 4/4] fix getting gcc major version

---
 build/tools/utils.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/tools/utils.py b/build/tools/utils.py
index f38dc5bda..255dd98f4 100644
--- a/build/tools/utils.py
+++ b/build/tools/utils.py
@@ -208,7 +208,7 @@ def get_gcc_major_version(gcc_path: str):
     capture_output=True,
     text=True,
   )
-  major_version = int(gcc_version_proc.stdout)
+  major_version = int(gcc_version_proc.stdout.split(".")[0])
 
   return major_version
 
